doc/Makefile

```
@@ -1,135 +1,27 @@
 # Makefile for Sphinx documentation
-#
 
 # You can set these variables from the command line.
-SPHINXOPTS    = -v
+SPHINXOPTS    = -nWT --keep-going
```

this is probably better: https://github.com/sphinx-gallery/sphinx-gallery/issues/301

```suggestion
SPHINXOPTS    = -vT
```